### PR TITLE
.github/ISSUE_TEMPLATE/bug_report.md: explicit mention that questions aren't appropriate for github issues

### PR DESCRIPTION
**Pull request by rouault**
Directly borrowed from GDAL bug_report.md

Admittedly this doesn't stop users abusing github issues, but at least we can
close such inappropriate tickets without much discussion

### CODE REVIEW

**Comment by roya0045** [comment]
Big +1

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -7,6 +7,12 @@ assignees: ''
 
 ---
 <!--
+IMPORTANT: Do NOT use GitHub to post any questions or support requests!
+           They will be closed immediately and ignored.
+
+Questions should go to the qgis-user mailing list at https://lists.osgeo.org/mailman/listinfo/qgis-user
+or other support forums. GitHub issues are for bug reports and suggestions for new features.
```
Comment (nyalldawson) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
or other support forums such as https://gis.stackexchange.com/. GitHub issues are for bug reports and suggestions for new features.
```

**Comment by Gustry** [comment]
Side question. It seems people are not reading too much the template (narrow text input, difficult to read...). 
Why not using the same approach as on QGIS-Documenation : 
Using a template like this : 
https://raw.githubusercontent.com/qgis/QGIS-Documentation/master/.github/ISSUE_TEMPLATE/2_application_issue.md which gives https://github.com/qgis/QGIS-Documentation/issues/new/choose It seems no-one is now using the QGIS Application Bug report button. Maybe @DelazJ or @havatv can confirm?

**Comment by roya0045** [comment]
> Side question. It seems people are not reading too much the template (narrow text input, difficult to read...).
> Why not using the same approach as on QGIS-Documenation :
> Using a template like this :
> https://raw.githubusercontent.com/qgis/QGIS-Documentation/master/.github/ISSUE_TEMPLATE/2_application_issue.md which gives https://github.com/qgis/QGIS-Documentation/issues/new/choose It seems no-one is now using the QGIS Application Bug report button. Maybe @DelazJ or @havatv can confirm?
> 
> It's written that users shouldn't create tickets about third party plugins, but we still have some tickets sometimes. Maybe we should do this too?

I think that more buttons would be a better option, an option could also be added for specific recurrent issues such as the mac 'malware' popup.

**Comment by m-kuhn** [comment]
@Gustry :100:

**Comment by Gustry** [comment]
I just made a PR with my proposition. #35017

**Pull request closed without merging**